### PR TITLE
Debian packaging, manpage, fixed static build

### PR DESCRIPTION
**Pull request by m-simonelli**
I have removed all the optimizing configure options in `buildscripts/linux-static/build.sh`, as I found that different versions of QT may or may not successfully compile using the exact same configure options. Note however, that this does increase compilation times.

### CODE REVIEW

**Review by chrisculnane — COMMENTED** [review]
I'm still looking into this. Removing the optimisations seems like a stop-gap and not what we want to do long term.

**Comment by m-simonelli** [comment]
Bump. As I said in the last comment I left, removing QT optimizations will not affect runtime performance - only compile time.

**Comment by chrisculnane** [comment]
What is the impact on the build size? If it is only a build time issue that isn't such an issue in the short term, however, if the build size is larger that might be an issue. Is [https://qtlite.com/](https://qtlite.com/) any help in constructing a working set of options?

**Comment by m-simonelli** [comment]
When I tested it the difference in binary size was only a few kilobytes. I'll have a look at QTLite

**Comment by m-simonelli** [comment]
QTLite *should* work, but we'd probably have to update the buildscripts on every new update of QT

**Comment by chrisculnane** [comment]
OK, let's leave it as is, we can look at QTLite before release. Can you resolve the merge conflicts on the debian-build branch and update or create a new pull request. The Master branch is now protected, so requires 2 approved reviews before it can be merged. Once the conflicts are resolved I'll approve and chase up another review.

**Comment by m-simonelli** [comment]
Alright, I'm just testing out the options that QTLite gave me so I might wait until that's done so I don't need to open a new PR later

File: buildscripts/linux-static/build-deps.sh [hunk 1]
```diff
@@ -22,12 +22,36 @@ pushd "$ROOT_SRC"
       git submodule update --init qtbase qtdeclarative qtgraphicaleffects qtimageformats qtquickcontrols qtsvg qtx11extras qttools qtmultimedia
       git submodule foreach --recursive git reset --hard
       git reset --hard
+      # TODO:
+      # Find which configure options can be included to compile QT without breaking the compilation
```
Comment (morganava) [inline buildscripts/linux-static/build-deps.sh:26]: this is great

**Review by morganava — APPROVED** [review]

**Review by chrisculnane — APPROVED** [review]

File: buildscripts/linux-static/build-deps.sh (hunk 1 above)
Comment (chrisculnane) [reply buildscripts/linux-static/build-deps.sh:26]: @pospeselr are you happy for me to merge this on master? Just wanted to check that isn't going to create any conflicts for you.

Comment (morganava) [reply buildscripts/linux-static/build-deps.sh:26]: Yeah it's good, doesn't block me at all

**Pull request merged**